src/errors.rs

```diff
@@ -43,9 +43,9 @@ pub async fn handle_rejection(err: warp::Rejection) -> Result<impl warp::Reply,
 /// Catch all error struct
 #[derive(Debug)]
 pub enum ReacherError {
-	DbError(sqlx::Error),
-	CsvError(),
-	JsonError(),
+	Db(sqlx::Error),
+	Csv(),
```

Comment:
What's the difference between this and just `Csv`?

Reply:
cargo clippy points this out as a an anti-pattern. Rather than having a common prefix, put the prefix in the enum name and use the qualified name everywhere as in use `ReacherError::Csv` to indicate it's an error of csv.